<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextPropertyGridWidget</class>
 <widget class="QWidget" name="TextPropertyGridWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>299</width>
     <height>179</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>299</width>
     <height>179</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="localizationKeyNameLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="localizationKeyTextLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="fontNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Font:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="fontSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>106</y>
      <width>57</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="textForAllStatesCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set text for all states</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fontSelectButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>96</y>
      <width>195</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align:right; font-size: 11px;</string>
    </property>
    <property name="text">
     <string>default font</string>
    </property>
   </widget>
   <widget class="QLabel" name="fontNameLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>148</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Font Color:</string>
    </property>
   </widget>
   <widget class="QColorButton" name="fontColorButton">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>146</y>
      <width>205</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QColorButton</class>
   <extends>QPushButton</extends>
   <header>qcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
